docs/Overview.md

```diff
@@ -0,0 +1,157 @@
+# TuxedoJS In a Nutshell
+>TuxedoJS is a client-side CommonJS based web development framework built on Facebook's awesome `React` view-layer and `Flux` Architecture. It provides a semantic interface for working with `Flux` and augmented `React` components for managing different aspects of the view logic.  Tux also provides some powerful prebuilt animation components that make animating transitions in `React`  a synch. Lastly, TuxedoJS leverages CommonJS modules to provide a completely modular framework which means, unlike other popular frameworks, you only load the pieces of Tux you actually intend to use.
+
+## The React View Layer
+TuxedoJS leverages `React` for its views.  `React` is Facebook's new high performance view library that provides really clean and semantic language for defining how your views should be rendered and when/how they should update. You can read more about `React` [here](http://facebook.github.io/react/).  `React` is a purely Javascript based view layer.  It uses a virtual DOM to compare the current DOM state to the new DOM state when updating so that it can submit only the minimal changes required to update the DOM. Lastly, `React` is often written using [JSX](http://facebook.github.io/react/docs/displaying-data.html) syntax, which allows you to write pseudo-html in Javascript.
```

Comment:
Maybe mention that changing the actual DOM is expensive and the virtual DOM allows for performance increases.